openedx/features/custom_fields/multiselect_with_other/forms/fields.py

```diff
@@ -2,8 +2,11 @@
 from django.utils.translation import ugettext_lazy as _
 from multiselectfield import MultiSelectFormField
 
-from openedx.features.custom_fields.multiselect_with_other.helpers import add_other_field_in_choices, get_other_values
-from openedx.features.custom_fields.multiselect_with_other.forms.widgets import CheckboxSelectMultipleWithOther, RadioSelectWithOther
+from openedx.features.custom_fields.multiselect_with_other.helpers import (
+    add_other_field_in_choices, get_other_values, filter_other_field_checkbox_value
+)
+from openedx.features.custom_fields.multiselect_with_other.forms.widgets import \
```

Comment:
Please avoid using \ here and other places as well if possible

Reply:
Regarding the import it'll automatically get resolved after running isort. Regarding other places in code, I also felt like we are unnecessarily breaking lines. Maybe let's only break lines if they exceed 120 lines or if readability is poor with a single line. I didn't comment earlier because I thought it is the coder's choice, but I will second @zeshanarifios here. Feel free to object @saad-waqar.